keyboards/fjlabs/vega/info.json

```diff
@@ -0,0 +1,126 @@
+{
+    "keyboard_name": "Vega",
+    "manufacturer": "FJLaboratories",
+    "url": "https://fjlaboratories.com/projects/",
+    "maintainer": "FJLaboratories",
+    "usb": {
+        "vid": "0x7074",
+        "pid": "0x0049",
+        "device_version": "0.0.1"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "rgblight": {
+        "led_count": 12,
+        "pin": "C7",
+        "hue_steps": 12,
+        "saturation_steps": 12,
+        "brightness_steps": 12,
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        }
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "matrix_pins": {
+        "cols": ["F0", "B1", "B2", "B3", "B7", "D0", "D1", "D2", "D3", "D5", "D7", "B4", "B5", "B6", "C6"],
+        "rows": ["F4", "F1", "F7", "F6", "F5"]
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
@lesshonor Thanks for the review on this one and the correction. Yes the board has multiple layouts so `LAYOUT_ALL` is the appropriate way of having it set. 

While I appreciate noroadsleft's efforts on my other keyboards, it hardly seems necessary to specify every permutation of layout (2 backspace, 2 caps, 3 bottom row = 12 layout permutations). It adds unnecessary bloat to the `info.json` **for a board that isn't being shipped without VIA support**. The primary intent of the board is to be shipped with VIA and all the layouts controlled and defined through there—not QMK Configurator. 

This is especially true given the additional redefinition involved in each one of these layouts, with only very minor changes in each. Easily adds 800 lines to the `info.json` for no substantial benefit. 

Let me know if you think otherwise.